deps.edn

```diff
@@ -76,7 +76,7 @@
   io.github.eerohele/pp                     {:git/tag "2024-01-04.60"           ; super fast pretty-printing library
                                              :git/sha "a428751"
                                              :git/url "https://github.com/eerohele/pp"}
-  io.github.metabase/macaw                  {:mvn/version "0.1.24"}             ; Parse native SQL queries
+  io.github.metabase/macaw                  {:mvn/version "0.1.26"}             ; Parse native SQL queries
```

Comment:
This reminds me of old school versioning where even is stable and odd is unstable 😄